CHANGELOG.md

```diff
@@ -5,11 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
-## [2.5.0.0.4]
+## [2.5.0.0.5]
 ### Added
 - Added configuration option `connections.antientropy_retry` (#4932)
 ### Changed
-- Set default antientropy_rety to run once per hour
+- Set default antientropy_rety to run once per hour (#4935)
```

Comment:
Typo -- antientropy_rety (should be retry)

Reply:
ea394b7e7c6136c90b90f8b983aa9a91ae4a0182